docs/core/interfaces/dialog.interface.md

```diff
@@ -48,6 +49,7 @@ interface DialogData {
 | actionsTemplate | `TemplateRef<any>` |    | Inserts a template styled on the left. Should be used for additional `mat-button` style buttons. (optional) |
 | descriptionTemplate | `TemplateRef<any>` |    | Inserts a description template. (optional) |
 | additionalActionButtons | `AdditionalDialogActionButton[]` |    | Inserts additional base-styled buttons into the action bar on the left. (optional) |
+| componentData | `any` |    | Data that injected in contentComponent. (optional) |
```

Comment:
should it be any?

Reply:
yes, because `DialogComponent` doesn't aware what data can be passed. This way you can pass whatever data you want to use to the `contentComponent`